data-reconciliation-app/src/endpoints/swagger.ts

```diff
@@ -0,0 +1,710 @@
+import * as ccfapp from "@microsoft/ccf-app";
+
+export function getSwaggerUI(): ccfapp.Response<string> {
+  /*
+  How to configure swagger UI to you application: https://swagger.io/docs/open-source-tools/swagger-ui/usage/installation/
+  */
+  return {
+    statusCode: 200,
+    headers: { "content-type": "text/html" },
+    body: `<!DOCTYPE html>
+    <html lang="en">
+      <head>
+        <meta charset="utf-8" />
+        <meta name="viewport" content="width=device-width, initial-scale=1" />
+        <meta name="description" content="SwaggerUI" />
+        <title>SwaggerUI</title>
+        <link rel="stylesheet" href="https://unpkg.com/swagger-ui-dist@4.5.0/swagger-ui.css" />
+        <script src="https://unpkg.com/swagger-ui-dist@4.5.0/swagger-ui-bundle.js" crossorigin></script>
+        <script src="https://unpkg.com/swagger-ui-dist@4.5.0/swagger-ui-standalone-preset.js" crossorigin></script>
+      </head>
+      <body>
+      <div id="swagger-ui"></div>
+      <script>
+        window.onload = () => {
+          window.ui = SwaggerUIBundle({
+            urls: [
+              {url: "/app/swagger.json", name: "Application API"}, /* '/app/api' will be used when ccf allow more control on openAPI document to be done through deployment bundle */
+              {url: "/gov/api", name: "Governance API"},
+              {url: "/node/api", name: "Operator API"}
+            ],
+            dom_id: '#swagger-ui',
+            presets: [SwaggerUIBundle.presets.apis,SwaggerUIStandalonePreset],
+            layout: "StandaloneLayout",
+          });
+        };
+      </script>
+      </body>
+    </html>
+    `
+  };
+
+}
+
+// return an OpenAPI document as the app documentation
+export function getOpenApiDocument(): ccfapp.Response<object> {
+  return {
+    statusCode: 200,
+    headers: { "content-type": "application/json" },
+    body: openApiDoc
+  };
+}
+
+
+const openApiDoc = {
```

Comment:
This feels like a part of the document that will change everytime someone adds an endpoint - is there a way to generate of keep this separate from what CCF gives us? This is probably best on a call!

Reply:
I've created a new Issue and added it's link in a comment